### PR TITLE
Image is not displaying the right way in the detail summary

### PR DESCRIPTION
**Pull request by vijay1189**

### CODE REVIEW

File: .ebextensions/nginx-upload-size.config [hunk 1]
```diff
@@ -2,3 +2,8 @@ files:
   /etc/nginx/conf.d/proxy.conf:
     content: |
       client_max_body_size 50M;
+
+
+container_commands:
+  01_reload_nginx:
+    command: "service nginx reload"
```
Comment (AlexanderUlitin) [inline .ebextensions/nginx-upload-size.config:9]: Can you please clarify when this command will run? On each deployment or setting up new server? I don't think we need it here, as I understand you only need to run it once after you changed config?

Comment (AlexanderUlitin) [reply .ebextensions/nginx-upload-size.config:9]: @vijay1189 can you please reply or fix it?

Comment (ramesh-solbeg) [reply .ebextensions/nginx-upload-size.config:9]: @AlexanderUlitin 
The file proxy.conf is copied from the unzipped deployment only after nginx is restarted. so after each deployment, we are restarting nginx manually using Container command, once we find a solution to only run once whenever a new instance is created we will modify this file.

Comment (AlexanderUlitin) [reply .ebextensions/nginx-upload-size.config:9]: @ramesh-solbeg ok, thanks.

**Pull request merged**